### ISSUE TITLE
regression in -next for arm64

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/continuous-integration#69]
```
aarch64-linux-gnu-ld: kernel/bpf/core.o: in function `bpf_get_prog_name':
/travis/linux-next/kernel/bpf/core.c:414: undefined reference to `btf_type_by_id'
aarch64-linux-gnu-ld: /travis/linux-next/kernel/bpf/core.c:414:(.text+0xce4): relocation truncated to fit: R_AARCH64_CALL26 against undefined symbol `btf_type_by_id'
aarch64-linux-gnu-ld: /travis/linux-next/kernel/bpf/core.c:415: undefined reference to `btf_name_by_offset'
aarch64-linux-gnu-ld: /travis/linux-next/kernel/bpf/core.c:415:(.text+0xcf8): relocation truncated to fit: R_AARCH64_CALL26 against undefined symbol `btf_name_by_offset'
```
https://travis-ci.com/ClangBuiltLinux/continuous-integration/builds/92111331

**Comment by nathanchance** [comment]
Fixed in the bpf tree, should be in fixed in the next -next version: https://git.kernel.org/pub/scm/linux/kernel/git/bpf/bpf-next.git/commit/?id=f6161a8f3036caa45f225486be39783e99e0fa29

**Comment by nathanchance** [comment]
Fixed in next-20181122: https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=f6161a8f3036caa45f225486be39783e99e0fa29
